fdbmonitor/CMakeLists.txt

```diff
@@ -35,3 +35,13 @@ add_custom_target(start_sandbox
                                              --lockfile ${CMAKE_BINARY_DIR}/sandbox/fdbmonitor.lock)
 
 add_dependencies(start_sandbox fdbmonitor fdbserver)
+
+if(NOT EXISTS ${CMAKE_BINARY_DIR}/generate_profile.sh)
+  configure_file(${CMAKE_SOURCE_DIR}/contrib/generate_profile.sh
```

Comment:
Why do we need to copy this file?

Reply:
copy the file from source dir to binary dir, I hear that this is the standard solution to run a script in cmake.

Reply:
Then, shouldn't we copy it to the `<build>/contrib`?

Reply:
done

Reply:
The first `if` check is checking the old path.
Also, isn't `CMAKE_BINARY_DIR` pointing `<build>/bin` directory?

Reply:
done. In line 34, "COMMAND ${CMAKE_BINARY_DIR}/bin/fdbmonitor ...", it seems CMAKE_BINARY_DIR just points to <build>